### PR TITLE
Add upgrade option to slcli hw.

### PR DESCRIPTION
**Pull request by FernandoOjeda**
Add upgrade option to slcli hw https://github.com/softlayer/softlayer-python/issues/1423.

I did not add the cpu option to be upgraded, because it seems to upgrade the cpu the portal ui creates a ticket to be upgraded manually.

### CODE REVIEW

**Review by caberos — APPROVED** [review]
LGTM

File: SoftLayer/managers/hardware.py [hunk 1]
```diff
@@ -780,6 +782,146 @@ def get_hardware_item_prices(self, location):
...
+        """
+        mask = [
+            'billingItem[id,package[id,keyName]]'
+        ]
+        mask = "mask[%s]" % ','.join(mask)
+
+        return self.hardware.getObject(id=instance_id, mask=mask)
+
+    def _get_upgrade_prices(self, instance_id, include_downgrade_options=True):
+        """Following Method gets all the price ids related to upgrading a VS.
```
Comment (ATGE) [inline SoftLayer/managers/hardware.py:861]: VS is for virtual server meaning,  should be HS for hardware server or just hardware I think.

File: SoftLayer/managers/hardware.py [hunk 2]
```diff
@@ -780,6 +782,146 @@ def get_hardware_item_prices(self, location):
...
+
+        :param int instance_id: Instance id of the VS to be upgraded
```
Comment (ATGE) [inline SoftLayer/managers/hardware.py:863]: VS ?

**Review by ATGE — COMMENTED** [review]
looks good, just a few comments.

File: SoftLayer/managers/hardware.py (hunk 1 above)
Comment (FernandoOjeda) [reply SoftLayer/managers/hardware.py]: Done.

File: SoftLayer/managers/hardware.py (hunk 2 above)
Comment (FernandoOjeda) [reply SoftLayer/managers/hardware.py]: Done.

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**